src/routes/api/invite/[token]/+server.ts

```diff
@@ -32,11 +37,24 @@ export const GET: RequestHandler = async ({ params, locals: { supabase, safeGetS
 		.select()
 		.single();
 	if (projectUserError || !projectUser) {
-		redirect(303, '/error');
+		error(500, 'joining_project_failure');
 	}
 
 	// Delete the invite
-	await supabase.from('project_invites').delete().eq('id', invite.id);
+	await supabaseAdmin.from('project_invites').delete().eq('id', invite.id);
 
 	redirect(303, `/dashboard/project/${invite.project_id}`);
 };
+
+const isUserAlreadyInProject = async (
+	supabase: SupabaseClient,
+	project_id: number,
+	user_id: string
+) => {
+	const { count } = await supabase
+		.from('project_users')
+		.select('*', { count: 'exact', head: true })
```

Comment:
```suggestion
		.select('id', { count: 'exact', head: true })
```
La query sera moins lourde